omero/export_scripts/Make_Movie.py

```diff
@@ -474,184 +468,183 @@ def writeMovie(commandArgs, conn):
+    pixels_id = pixels.getId()
+
+    size_x = pixels.getSizeX()
+    size_y = pixels.getSizeY()
+    size_z = pixels.getSizeZ()
+    size_c = pixels.getSizeC()
+    size_t = pixels.getSizeT()
+
+    if (size_x is None or size_y is None or size_z is None or size_t is None or
+            size_c is None):
         return
 
     if (pixels.getPhysicalSizeX() is None):
-        commandArgs["Scalebar"] = 0
-
-    cRange = range(0, sizeC)
-    cWindows = None
-    cColours = None
-    if "ChannelsExtended" in commandArgs and \
-            validChannels(commandArgs["ChannelsExtended"], sizeC):
-        cRange = []
-        cWindows = []
-        cColours = []
-        for c in commandArgs["ChannelsExtended"]:
+        command_args["Scalebar"] = 0
+
+    c_range = range(0, size_c)
+    c_windows = None
+    c_colours = None
+    if "ChannelsExtended" in command_args and \
+            valid_channels(command_args["ChannelsExtended"], size_c):
+        c_range = []
+        c_windows = []
+        c_colours = []
+        for c in command_args["ChannelsExtended"]:
             m = re.match('^(?P<i>\d+)(\|(?P<ws>\d+)' +
                          '\:(?P<we>\d+))?(\$(?P<c>.+))?$', c)
             if m is not None:
-                cRange.append(int(m.group('i'))-1)
-                cWindows.append([float(m.group('ws')), float(m.group('we'))])
-                cColours.append(m.group('c'))
-    elif "Channels" in commandArgs and \
-            validChannels(commandArgs["Channels"], sizeC):
-        cRange = commandArgs["Channels"]
-
-    tzList = calculateRanges(sizeZ, sizeT, commandArgs)
-
-    timeMap = calculateAquisitionTime(conn, pixelsId, cRange, tzList)
-    if (timeMap is None):
-        commandArgs["Show_Time"] = False
-    if (timeMap is not None):
-        if (len(timeMap) == 0):
-            commandArgs["Show_Time"] = False
-
-    frameNo = 1
-    omeroImage.setActiveChannels(map(lambda x: x+1, cRange),
-                                 cWindows,
-                                 cColours)
-    renderingEngine = omeroImage._re
-
-    overlayColour = (255, 255, 255)
-    if "Overlay_Colour" in commandArgs:
-        r, g, b, a = COLOURS[commandArgs["Overlay_Colour"]]
-        overlayColour = (r, g, b)
-
-    canvasColour = tuple(COLOURS[commandArgs["Canvas_Colour"]][:3])
-    mw = commandArgs["Min_Width"]
-    if mw < sizeX:
-        mw = sizeX
-    mh = commandArgs["Min_Height"]
-    if mh < sizeY:
-        mh = sizeY
+                c_range.append(int(m.group('i'))-1)
+                c_windows.append([float(m.group('ws')), float(m.group('we'))])
+                c_colours.append(m.group('c'))
+    elif "Channels" in command_args and \
+            valid_channels(command_args["Channels"], size_c):
+        c_range = command_args["Channels"]
+
+    tz_list = calculate_ranges(size_z, size_t, command_args)
+
+    time_map = calculate_acquisition_time(conn, pixels_id, c_range, tz_list)
+    if (time_map is None):
```

Comment:
Due to installation issue, we might actually have to review the script and find another way to generate movie